source/options.css

```diff
@@ -16,15 +16,29 @@ ul {
 	padding-left: 1.5em;
 }
 
+[data-scope] {
+	float: right;
+	margin-right: 50px;
+}
```

Comment:
My idea was that the bullet points themselves can be replaced without causing any layout shifts. This is what I currently see: (ignore the token)

<img width="510" alt="" src="https://user-images.githubusercontent.com/1402241/100534857-18487c80-31d9-11eb-91c3-0141eb57a4a5.png">

Reply:
What browser? I tried in both Firefox and chrome.

Reply:
Chrome